stdata/vis/spacetime.py

```diff
@@ -348,7 +348,13 @@ def update_active(self, _id):
 
 class SpaceTimeVisualise(object):
     def __init__(
-        self, train_df, test_df, sat_df=None, geopandas_flag=True, test_start=None
+        self,
+        train_df,
+        test_df,
+        train_end,
```

Comment:
thanks for looking into this. Why cant you use  `test_start`? i think this should already plot a line where the testing time starts